src/editor/esphome-editor.ts

```diff
@@ -61,6 +61,8 @@ export class ESPHomeEditor extends LitElement {
         enabled: false,
       },
       tabSize: 2,
+      fontFamily:
+        "SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace",
```

Comment:
Add Roboto Mono to it.

Reply:
Try this https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/#the-monospaced-system-ui-css-font-stack

Reply:
Let's just copy what Tailwind uses

https://github.com/tailwindlabs/tailwindcss/blob/master/src/css/preflight.css#L106

Reply:
Also don't wrap the whole value in quotes, that is not supported

Reply:
too many quotes there

Reply:
> Also don't wrap the whole value in quotes, that is not supported

this is not a css style rule, it's a js object